src/backend/dx11/src/lib.rs

```diff
@@ -59,19 +59,18 @@ impl Instance {
     #[doc(hidden)]
     pub fn create() -> Self {
         // Create DXGI factory
-        let mut dxgi_factory = ComPtr::<winapi::IDXGIFactory2>::new(ptr::null_mut());
-
+        let mut dxgi_factory: *mut winapi::IDXGIFactory2 = ptr::null_mut();
         let hr = unsafe {
             dxgi::CreateDXGIFactory1(
                 &dxguid::IID_IDXGIFactory2,
-                dxgi_factory.as_mut() as *mut *mut _ as *mut *mut c_void)
```

Comment:
Sry, I can't follow as there is already a `as_mut` in wio's `ComPtr` and I don't see how/where a new method would help.
The only thing I missed a bit was a function returning `&mut *mut T` which resulted in first creating a pointer and then store it later into the `ComPtr` wrapper.